ts/parts/PieSeries.ts

```diff
@@ -1051,29 +1051,37 @@ seriesType<Highcharts.PieSeries>(
         drawEmpty: function (this: Highcharts.PieSeries): void {
             var centerX,
                 centerY,
+                start = this.startAngleRad,
+                end = this.endAngleRad,
                 options = this.options;
 
             // Draw auxiliary graph if there're no visible points.
             if (this.total === 0) {
                 centerX = this.center[0];
                 centerY = this.center[1];
 
-                if (!this.graph) { // Auxiliary graph doesn't exist yet.
-                    this.graph = this.chart.renderer.circle(centerX,
-                        centerY, 0)
+                if (!this.graph) {
+                    this.graph = this.chart.renderer
+                        .arc(centerX, centerY, this.center[1] / 2, 0, start, end)
                         .addClass('highcharts-graph')
                         .add(this.group);
                 }
 
-                this.graph.animate({
+                this.graph.attr({
+                    d: Highcharts.SVGRenderer.prototype.symbols.arc(
+                        centerX,
+                        centerY,
+                        this.center[2] / 2,
+                        0, {
+                            start: start,
+                            end: end,
+                            innerR: options.innerSize
+                        }
+                    ),
                     'stroke-width': options.borderWidth,
-                    cx: centerX,
-                    cy: centerY,
-                    r: this.center[2] / 2,
-                    fill: (options.fillColor as any) || 'none',
-                    stroke: (options.color as any) ||
-                        '${palette.neutralColor20}'
-                }, this.options.animation);
+                    fill: options.fillColor || 'none',
+                    stroke: options.color || '#cccccc'
```

Comment:
One change that has slipped through into production already, but we should fix it now that we are here. 

1. These presentational properties should not apply in `styledMode`. See https://jsfiddle.net/highcharts/n9s2Ld1a/. 
2. For the default stroke color, use `${palette.neutralColor20}`
3. Add a more specific class name to the graph, suggesting `highcharts-empty-series`.